### ISSUE TITLE
Custom input issue

### ISSUE THREAD

**Issue by curiousQuestions** [Janspiry/Image-Super-Resolution-via-Iterative-Refinement#40]
Please pardon my inexperience using SR GANs, I wondering if you could clarify what is the role of the data input folders:
1. Should the` l_resolution`/`h_resolution` folders contain the same images (with different sizes)? I assume so for testing/validation steps. 
2. What sort of images does the `sr_16_64` contains initially?
3. Where should I put the images for which I want to increase the resolution?

Thank you in advance for your answer and thank you for sharing your awesome work!

**Comment by Janspiry** [comment]
Thanks for your attention. The input folder is like below:
![image](https://user-images.githubusercontent.com/32922494/147909570-860503fb-09fd-4924-a9bd-094763467827.png)

Take the `sr_16_64` as a example, You should put the 16×16 image (ground truth or downsample images) to the `lr_16` fold, then use the 'prepare.py' to upsample it and put them `sr_16_128` fold. 

The model use `sr_16_128` fold to increase the resolution. And the `hr_128` is not necessary if you don't have 128×128 ground truth images (for validation), you can just copy it from `sr_16_128` fold.

**Comment by curiousQuestions** [comment]
This is very helpful, thank you for your time and your clarification! 
I used`/data/prepare_data.py` to prepare the upsampled version of the 16x16 images but the output was of `.mdb` type. Is it possible to use create a folder containing `.png` images directly using your script? Otherwise I will simply run a bicubic interpolation script locally.
My understanding is that the above steps are related to the training phase. Once the model is trained, how can I use it to upsample a specific image? In our example above, I would use `python infer.py -c sr_sr3_16_128.json` and put the 128x128 images I want to upsample in `sr_16_128`. Would this be correct?

**Comment by Janspiry** [comment]
![image](https://user-images.githubusercontent.com/32922494/148012836-4fa65bee-b52c-46b4-83ca-4df215627cf9.png)
1. As shows, you can choose the `img` input type.
2. You can right, just put the 128x128 images in sr_16_128 folder. More details can be found in README `Own Data` part.

**Comment by curiousQuestions** [comment]
Once again, thank you so much for your precisions!
I ended up having an OOM issue so I could not run the model but this is a set up I need to fix on my side

**Comment by masuxin** [comment]
Sorry I don't understand this process very well, I see that images of the validation set are also placed in the folder like this, if you need to put a 128*128 image in the hr_128 during verification, then where is the super resolution reflected? Can this model output a high-resolution(eg.128*128) image to any low-resolution(eg.16*16) image?
Thank you in advance for your answer!
